cmsplugin_articles_ai/models/articles.py

```diff
@@ -153,6 +153,9 @@ class Article(PublisherModel):
     modified_at = models.DateTimeField(
         _("last modified"), auto_now=True, editable=False,
     )
+    show_lead = models.BooleanField(
+        _("show lead"), default=True, help_text=_("Show lead text."),
```

Comment:
This article model doesn't have any lead text specified. Other projects tough need this functionality which are using this django app. Example use case:
```
{% if article.show_lead %}
            <p>{{ article.main_content|striptags|truncatechars:80 }}</p>
{% endif %}
```